aiosmtpd/docs/conf.py

```diff
@@ -148,6 +150,12 @@ def syspath_insert(pth: Path):
     "python": ("https://docs.python.org/3", None),
 }
 
+doctest_global_setup = """
+import sys
+in_win32 = sys.platform == "win32"
+in_cygwin = sys.platform == "cygwin"
+"""
```

Comment:
Need to check for Cygwin as well (UnixSocketController does not work properly on Cygwin)